biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,22 @@
+# Biotope is a framework and architecture which enables you to develop technology and cms independent frontend component libraries
+
+
+## It helps developers to:
+- produce clean and state of the art frontend code
+- develope ui-components on a unified standard and with latest web standards
+- keep focus on developing components, the architecture and dev environment is provided by Biotope.
```

Comment:
- keep focus on component logic in opposition to creating an environment to make your code run